### PR TITLE
HS-470: Add MonitoredService into Inventory

### PR DESCRIPTION
**Pull request by james8hut**
## Description
Add MonitoredService into Inventory

## Jira link(s)
- https://issues.opennms.org/browse/HS-470

## Flagged for review
none

## Checklist
* [ ] Follows Horizon Stream's [development guidelines.](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS/horizon-stream/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.

### CODE REVIEW

File: inventory/src/main/java/org/opennms/horizon/inventory/rest/MonitoredServiceRest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/inventory/services")
```
Comment (thomasbigger584) [inline inventory/src/main/java/org/opennms/horizon/inventory/rest/MonitoredServiceRest.java:20]: should `/inventory` be a spring context path instead if its consistently the same for each rest endpoint in the inventory service?

Comment (james8hut) [reply inventory/src/main/java/org/opennms/horizon/inventory/rest/MonitoredServiceRest.java:20]: Possibly, but the REST layer is probably going away.

Comment (james8hut) [reply inventory/src/main/java/org/opennms/horizon/inventory/rest/MonitoredServiceRest.java:20]: Hmm, after trying this, I would probably prefer not. If you do it in application.yml, then it needs to be in both src/main/resources and src/test/resources. As such, if there is a different value in main than test, your tests don't really do their job. (Just placing it in main causes the Junit tests to fail.)

https://www.baeldung.com/spring-boot-context-path

Possibly the most workable place to put it is the main method. But that doesn't strike me as a great place to have it. Seems hidden.

Comment (thomasbigger584) [reply inventory/src/main/java/org/opennms/horizon/inventory/rest/MonitoredServiceRest.java:20]: Another option is to use a test profile in your tests, such that the two application.ymls get merged

**Pull request merged**